aws/ami/files/scylla_install_ami

```diff
@@ -167,3 +167,22 @@ if __name__ == '__main__':
                 f.write(u'exclude=kernel kernel-devel')
         elif platform.machine() == 'aarch64':
             run('yum -y --enablerepo=centos-kernel update kernel')
+
+    if distro == 'centos':
```

Comment:
This will fail on RHEL and other derivatives.  We always created the AMI in the past on CentOS, but better to be prepared.

Reply:
Well, distribution detection code on scylla_install_ami is much simplified than scylla setup scripts, it just checking existance of /etc/redhat-release, and detect like this:
if exists -> 'centos' (= RHEL based distro) / if not exists -> 'ubuntu' (= non-RHEL distro).
So it actually works with RHEL & OEL.

To get more specific distribution information we need to parse /etc/os-release, but I think this is enough for current implementation.